apache/trafficcontrol
# Validate ORG server for topology based DS

## What does this PR (Pull Request) do?




## Which Traffic Control components are affected by this PR?


- Traffic Ops

## What is the best way to verify this PR?


1. Remove ORG Server from a given DS's topology
2. Try and update a DS and it should not allow you to do it.

Also, run the api/v3 test cases.

## If this is a bug fix, what versions of Traffic Control are affected?


## The following criteria are ALL met by this PR




## Additional Information

API route hasn't changed. Hence the documentation is not updated.
